vpr/src/base/vpr_types.h

```diff
@@ -1029,6 +1035,9 @@ enum class e_incr_reroute_delay_ripup {
 constexpr int NO_FIXED_CHANNEL_WIDTH = -1;
 
 struct t_router_opts {
+    e_node_reorder_algorithm reorder_rr_graph_nodes_algorithm = DONT_REORDER;
```

Comment:
I'd move these to the end of the router_opts as they're not going to be the most commonly used options, and add a comment in front of the group of 3 saying they control rr-graph node reordering, which can be useful for testing and could optimize memory layout (but hasn't so far).

Reply:
Done.